### PR TITLE
Quick Start - Grow List: Fix Text wrapping

### LINKED ISSUE #15182: Quick Start - Grow List: Text wrap seems a little off
### Expected behavior
Based on the mockup in https://github.com/wordpress-mobile/WordPress-Android/pull/15052,  I expected text wrap to kick in when the screen width was smaller, or wrap only one word.

### Actual behavior
"Keep up to date on your site's performance" seems to wrap a little early, and wraps the last two words instead of only one.

image

### Steps to reproduce the behavior
1. Create a new site.
2. Choose "Show me around" on the Quick Start prompt.
3. Tap "Grow your audience" to open the list of tasks.
4. Compare the subtext under "Check your stats" to that in https://href.li/?https://github.com/wordpress-mobile/WordPress-Android/pull/15052 

Not sure if this is actually problematic, but it looked visually off.

### PR DESCRIPTION
**Pull request by ashiagr**
Fixes #15182

This PR allows [widow words](https://www.proofreadingacademy.com/advice/editing-tips-widows-and-orphans-explained/) or the "dangling one word" for quick start subtitles to fix visually off text-wrapping.

To test:

1. Create a new site.
2. Choose "Show me around" on the Quick Start prompt.
3. Tap "Grow your audience" to open the list of tasks.
4. Compare the subtext under "Check your stats" 

Before | After
-------|------
widow-word-fix | widow-word-fix-removed

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by zwarm — APPROVED** [review]
Looks good @ashiagr. Just waiting on design review before merging.

**Review by osullivanchris — APPROVED** [review]
Looks as expected. So as I understand, we had some code in their to prevent widow words before? But now we're removing it, so that the text visually looks more balanced?

As its detected in code, I also assume it behaves the same in all languages.

**Comment by ashiagr** [comment]
> Looks as expected. So as I understand, we had some code in their to prevent widow words before? But now we're removing it, so that the text visually looks more balanced?

Correct

> As its detected in code, I also assume it behaves the same in all languages.

Removing widow words is a custom logic (convert last space to non-breaking space so there's no dangling one word to text views where it is applied) added as part of this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/1972

I see that not all text views in the code apply it. It makes sense for paragraphs to not have widow words. But in this case. for sub-titles, I think we can allow them for a better visual effect.

**Comment by osullivanchris** [comment]
@ashiagr thanks for the explanation!

Pretty cool capability to do it. But I agree in this case the line is so short, it looks better without this custom code.

Nice job

**Pull request merged**
